src/sentry/snuba/metrics/datasource.py

```diff
@@ -660,7 +639,8 @@ def get_all_tags(
             start=start,
             end=end,
         )
-    except InvalidParams:
+    except InvalidParams as e:
+        sentry_sdk.capture_exception(e)
```

Comment:
Added the exception capturing for us to be aware of any issues. Didn't want to refactor behavior to increase the scope of the PR.